tests/queries/0_stateless/02541_lightweight_delete_on_cluster.sql

```diff
@@ -0,0 +1,23 @@
+-- Tags: distributed, no-replicated-database
+-- Tag no-replicated-database: ON CLUSTER is not allowed
```

Comment:
Can we add separate test for Database replicated? How it works in that case - it is always "ON CLUSTER"?